### ISSUE TITLE
Limit for downloading orders from Allegro

### ISSUE THREAD

**Issue by TarnJul** [allegro/allegro-api#10412]
Hello. Is there any limit for downloading orders from Allegro? Our manager noticed that more than 100 orders are not loaded

**Comment by MaciejFrackowiak** [comment]
No, we don't have such a limit. You can retrieve orders from the last 365 days.
